case-lib/lib.sh

```diff
@@ -253,6 +253,50 @@ func_lib_get_tplg_path()
     return 0
 }
 
+# We must not quote SOF_ALSA_OPTS and disable SC2086 below for two reasons:
+#
+# 1. We want to support multiple parameters in a single variable, in
+#    other words we want to support this:
+#
+#      SOF_ALSA_OPTS="--foo --bar"
+#      aplay $SOF_ALSA_OPTS ...
+#
+# 2. aplay does not ignore empty arguments anyway, in other words this
+#    does not work anyway:
+#
+#      SOF_ALSA_OPTS=""
+#      aplay "$SOF_ALSA_OPTS" ...
+#
+# This is technically incorrect because it means our SOF_ALSA_OPTS
+# cannot support whitespace, for instance this would be split in two
+# options: --bar="aaaa and bbbb"
+#
+#      SOF_ALSA_OPTS='--foo --bar="aaaa bbbb"'
+#
+# To do this "correctly" SOF_ALSA_OPTS should be an array.
+# From https://mywiki.wooledge.org/BashGuide/Arrays "The only safe way
+# to represent multiple string elements in Bash is through the use of
+# arrays."
+# However 1. it would complicate the user interface 2. We don't seem to
+# need arguments with whitespace or globbing characters.
+
+# Default value, can be overriden by the environment. Empty string does
+# override.
+: "${SOF_ALSA_OPTS=--fatal-errors}"
+
+aplay_opts()
```

Comment:
I think `opts` means options? maybe a more good name for the wrapper function

Reply:
I've been looking for a better but still short name but I didn't find any, suggestions welcome. It has  to be  kept relatively short because it will be used a lot.

Reply:
Same feeling here. If I had a better name, I should type it already.